modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -79,7 +96,7 @@ type HTTPTransport struct {
 	// connection. A negative value disables this. Default: `300ms`.
 	FallbackDelay caddy.Duration `json:"dial_fallback_delay,omitempty"`
 
-	// How long to wait for reading response headers from server. Default: No timeout.
+	// How long to wait/Roundtrip for reading response headers from server. Default: No timeout.
```

Comment:
I'm not sure why `/Rountrip` was added, doesn't really add anything. I think it was probably a weird copy-paste mistake of somekind with your cursor sitting in the middle of nowhere :sweat_smile: 

```suggestion
	// How long to wait for reading response headers from server. Default: No timeout.
```